src/data.js

```diff
@@ -1,12 +1,16 @@
 export function fetchBreeds() {
-  return fetch("https://api.thedogapi.com/v1/breeds/")
+  return fetch("https://api.thedogapi.com1/v1/breeds/")
```

Comment:
je to pravilen url `https://api.thedogapi.com1/v1/breeds/` -> com1.. a typo?

Reply:
typo je, hotel sem `catch error `stestirati